dts/bindings/tach/nuvoton,npcx-tach.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-Tachometer node
+
+compatible: "nuvoton,npcx-tach"
+
+include: tach.yaml
+
+properties:
+    reg:
+        required: true
+    clocks:
+        required: true
+    pinctrl-0:
+        type: phandles
+        required: false
+        description: configurations of pinmux controllers in tachometers
+    sample_clk:
+        type: int
+        required: false
+        description: |
+          sampling clock frequency of tachometer. Please notice that it must be
+          fixed to 32768 if bus in clocks property is NPCX_CLOCK_BUS_LFCLK.
+    port:
+        type: int
+        required: false
+        description: selected port of tachometer (port-A is 0 and port-B is 1)
+    pulses_per_round:
+        type: int
+        required: false
```

Comment:
It looks to me like these last three should be required, since you will get a compile error otherwise, I think? These values are used in tach_cfg_##inst above

Reply:
Since all default statuses of tachometers are "disabled", it won't get a compile error. unless you set their statuses to 'okay' in the board's dts file. The reason that the "required" property is false is the users need to set these properties explicitly in dts file. Or you will get a compiler error. 

If we set  "required" as true for these properties, you need to set the default value for them in soc's dtsi file. It might cause ambiguity if someone forgot to set the correct value of these properties in board's dts file.

Reply:
Gosh I didn't know that. What a pain!